### ISSUE TITLE
AI broke Schedule page

### ISSUE THREAD

**Issue by kant2002** [terrajobst/apireview.net#3]
As unauthenticated person, I go to Request page and nothing happens. (Actually Yellow Blazor Bar appears). In the console log
```
[2022-06-09T12:42:25.643Z] Information: Normalizing '_blazor' to 'https://apireview.net/_blazor'.
apireview.net/:1 
        
       Tracking Prevention blocked access to storage for https://m.youtube.com/static/r/efd3a499/scheduler.vflset/scheduler.js?id=1&udid=d7fdc40a617b4e7c&config=eyJhY3RpdmUiOnRydWUsInN0YXJ0IjoiaHR0cHM6Ly9tLnlvdXR1YmUuY29tL3N0YXRpYy9yL2VmZDNhNDk5L3NjaGVkdWxlci52ZmxzZXQvc2NoZWR1bGVyLmpzIiwiY291bnQiOjEsInByb2QiOnRydWUsInRpbWVyIjoxNjAwMDAsInRpbWUiOjQyfQ==.
apireview.net/:1 
        
       Tracking Prevention blocked access to storage for https://m.youtube.com/?app=m.
blazor.server.js:1 [2022-06-09T12:42:26.705Z] Information: WebSocket connected to wss://apireview.net/_blazor?id=234Yv3mkmn1VJxF7oS10vQ.
apireview.net/:1 
        
       Tracking Prevention blocked access to storage for https://m.youtube.com/.
blazor.server.js:1 
        
       [2022-06-09T12:42:34.197Z] Error: There was an unhandled exception on the current circuit, so this circuit will be terminated. For more details turn on detailed exceptions by setting 'DetailedErrors: true' in 'appSettings.Development.json' or set 'CircuitOptions.DetailedErrors'. 
log @ blazor.server.js:1
blazor.server.js:1 [2022-06-09T12:42:34.198Z] Information: Connection disconnected.
4blazor.server.js:1 
        
       Uncaught (in promise) Error: Cannot send data if the connection is not in the 'Connected' State.
    at bt.send (blazor.server.js:1:51876)
    at kt._sendMessage (blazor.server.js:1:68532)
    at kt._sendWithProtocol (blazor.server.js:1:68573)
    at kt.send (blazor.server.js:1:68681)
    at blazor.server.js:1:128406
    at be (blazor.server.js:1:27771)
    at ve (blazor.server.js:1:27734)
    at blazor.server.js:1:19884
    at blazor.server.js:1:13427
    at Array.forEach (<anonymous>)

```
I was manage to authenticate and press Request again, with same effect. After some time page start working magically.

**Comment by terrajobst** [comment]
Hmm, might have been a temporary issue during deployment? Request is basically a static page where nothing interesting happens...

Does this repro for you or was this just a temporary glitch?

**Comment by kant2002** [comment]
I receive this error two times with 2-5 minutes difference. And now I go to double-check and crash Blazor by going from Request to Schedule. Seems Schedule chrash consistently for not logged user. No error in the Network tab. That's probably unrelated error.

**Comment by kant2002** [comment]
```
Uncaught (in promise) Error: Cannot send data if the connection is not in the 'Connected' State.
    at bt.send (blazor.server.js:1:51876)
    at kt._sendMessage (blazor.server.js:1:68532)
    at kt._sendWithProtocol (blazor.server.js:1:68573)
    at kt.send (blazor.server.js:1:68681)
    at blazor.server.js:1:128406
    at be (blazor.server.js:1:27771)
    at ve (blazor.server.js:1:27734)
    at blazor.server.js:1:19884
    at blazor.server.js:1:13427
    at Array.forEach (<anonymous>)
```

**Comment by kant2002** [comment]
OOh, maybe the issue on Schedule page all the time an I just click too fast to realize that.

**Comment by terrajobst** [comment]
Oh yes. The scheduling seems busted. Odd. LMTAL.

**Comment by terrajobst** [comment]
Fixed with 267c4680db856a10b1dd3de3ce7eb3c92f6c8127
